spec/acts_as_tenant/model_extensions_spec.rb

```diff
@@ -199,14 +199,46 @@
   end
 
   describe "It should be possible to use associations with foreign_key from polymorphic" do
-    before do
-      @account = Account.create!(name: 'foo')
-      ActsAsTenant.current_tenant = @account
-      @project = Project.create!(name: 'project', account: @account)
-      @comment = Comment.new commentable: @project, account: @account
+    context 'tenanted objects have a polymorphic association' do
+      before do
+        @account = Account.create!(name: 'foo')
+        ActsAsTenant.current_tenant = @account
+        @project = Project.create!(name: 'project', account: @account)
+        @comment = Comment.new commentable: @project, account: @account
+      end
+
+      it { expect(@comment.save!).to eq(true) }
     end
 
-    it { expect(@comment.save!).to eq(true) }
+    context 'tenant is polymorphic' do
+      before do
+        @account = Account.create!(name: 'foo')
+        @project = Project.create!(name: 'polymorphic project')
+        ActsAsTenant.current_tenant = @project
+        @comment = PolymorphicTenantComment.new(account: @account)
+      end
+
+      it 'populates commentable_type with the current tenant' do
+        expect(@comment.polymorphic_tenant_commentable_id).to eql(@project.id)
+        expect(@comment.polymorphic_tenant_commentable_type).to eql(@project.class.to_s)
+      end

```

Comment:
Without the patch, this test would not populate the polymorphic type column. You would get the following test failure:
```
             @comment = PolymorphicTenantComment.new(account: @account)
           end
     
           it 'populates commentable_type with the current tenant' do             
 =>          expect(@comment.commentable_type).to eql(@project.class.to_s)
             expect(@comment.commentable_id).to eql(@project.id)
           end
         end
       end
     

[1] pry(#<RSpec::ExampleGroups::ActsAsTenant::ItShouldBePossibleToUseAssociationsWithForeignKeyFromPolymorphic::TenantIsPolymorphic>)> @comment
=> #<PolymorphicTenantComment:0x007f8b5641cf10 id: nil, commentable_id: 1, commentable_type: nil, account_id: 1>
[2] pry(#<RSpec::ExampleGroups::ActsAsTenant::ItShouldBePossibleToUseAssociationsWithForeignKeyFromPolymorphic::TenantIsPolymorphic>)> exit
F

Failures:

  1) ActsAsTenant It should be possible to use associations with foreign_key from polymorphic tenant is polymorphic populates commentable_type with the current tenant
     Failure/Error: expect(@comment.commentable_type).to eql(@project.class.to_s)
     
       expected: "Project"
            got: nil
     
       (compared using eql?)
     # ./spec/acts_as_tenant/model_extensions_spec.rb:223:in `block (4 levels) in <top (required)>'

Finished in 24.14 seconds (files took 1.37 seconds to load)
1 example, 1 failure

Failed examples:

rspec ./spec/acts_as_tenant/model_extensions_spec.rb:221 # ActsAsTenant It should be possible to use associations with foreign_key from polymorphic tenant is polymorphic populates commentable_type with the current tenant

```